airflow-core/src/airflow/api_fastapi/execution_api/routes/task_instances.py

```diff
@@ -53,7 +55,9 @@
 from airflow.utils import timezone
 from airflow.utils.state import DagRunState, TaskInstanceState
 
-router = VersionedAPIRouter(
+router = VersionedAPIRouter()
+
+ti_id_router = VersionedAPIRouter(
```

Comment:
Can we add a comment here explaining why we need this one? It will be clear to reader too